Extractor/Config.cs

```diff
@@ -642,6 +642,23 @@ public class SubscriptionConfig
         /// Log bad subscription datapoints
         /// </summary>
         public bool LogBadValues { get; set; } = true;
+
+        /// <summary>
+        /// Server namespace nodes the should be subscribed to a rebrowse upon changes to their values.
+        /// </summary>
+        public ServerNamespacesToRebrowseConfig? ServerNamespacesToRebrowse { get; set; }
+    }
+    public class ServerNamespacesToRebrowseConfig
+    {
+        /// <summary>
+        /// Indicate if the extractor should turn on subscription to namespaces.
+        /// </summary>
+        public bool Subscribe { get; set; } = false;
+        /// <summary>
+        /// Desired namespace nodes to subscribe to.
```

Comment:
And then have this be optional. Be aware that setting it equal to something here does not prevent a user from setting it to null in config, so you can't trust the compiler now telling you that it is non-null.